fourmolu.yaml

```diff
@@ -0,0 +1,70 @@
+# Fourmolu configuration file
+
+# Further description on these options can be found at https://fourmolu.github.io/config which also lists useful examples.
+
+# Number of spaces per indentation step
+indentation: 2
+
+# Max line length for automatic line breaking
+column-limit: 120
+
+# Styling of arrows in type signatures (choices: trailing, leading, or leading-args)
+function-arrows: leading
+
+# How to place commas in multi-line lists, records, etc. (choices: leading or trailing)
+comma-style: leading
+
+# Styling of import/export lists (choices: leading, trailing, or diff-friendly)
+import-export-style: diff-friendly
+
+# Rules for grouping import declarations
+import-grouping: by-scope
+
+# Whether to full-indent or half-indent 'where' bindings past the preceding body
+indent-wheres: false
+
+# Whether to leave a space before an opening record brace
+record-brace-space: true
+
+# Number of spaces between top-level declarations
+newlines-between-decls: 1
+
+# How to print Haddock comments (choices: single-line, multi-line, or multi-line-compact)
+haddock-style: single-line
+
+# How to print module docstring
+haddock-style-module: single-line
+
+# Styling of let blocks (choices: auto, inline, newline, or mixed)
+let-style: auto
+
+# How to align the 'in' keyword with respect to the 'let' keyword (choices: left-align, right-align, or no-space)
+in-style: right-align
+
+# Whether to put parentheses around a single constraint (choices: auto, always, or never)
+single-constraint-parens: never
+
+# Whether to put parentheses around a single deriving class (choices: auto, always, or never)
+single-deriving-parens: never
+
+# Output Unicode syntax (choices: detect, always, or never)
+unicode: detect
+
+# Give the programmer more choice on where to insert blank lines
+respectful: false
+
+# Fixity information for operators
+fixities: []
+
+# Module reexports Fourmolu should know about
+reexports: []
+
+# This option determines whether classes in deriving clauses should be sorted alphabetically or left alone.
+sort-derived-classes: true
+
+# This option determines whether deriving clauses should be sorted alphabetically or left alone.
+sort-derived-clauses: true
+
+# This option determines whether constraints should be sorted alphabetically or left alone.
+# We have set it to false for now since order of constraints lead to implicit ordering of type variables, changing it now would break existing code.
```

Comment:
A better practice would be to fix the order with `forall ... .` declaration, but I think it's better to be left to the separate PR since it changes semantics of the code